libr/include/r_bind.h

```diff
@@ -39,6 +43,8 @@ typedef struct r_core_bind_t {
 	RCoreConfigGet cfgGet;
 	RCoreNumGet numGet;
 	RCoreIsMapped isMapped;
+	RCoreDebugMapsGet getDebugMaps;
```

Comment:
Both callbacks are unnecessary

Reply:
syncDebugMaps is needed to sync only once instead of syncing in each isMapped call.